testdata/testdir2/wom_bat/cur/atomic

```diff
@@ -4,8 +4,8 @@ Subject: atoms
 To: "Democritus" <demo@example.com>
 Message-id: <3BE9E6535E302944823E7A1A20D852173@msg.id>
 MIME-version: 1.0
-Content-type: text/plain; charset=us-ascii
-Content-transfer-encoding: 7BIT
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
 Precedence: high
 
```

Comment:
Is this change needed?

Reply:
Yes, otherwise the `msg:body` test fails, because with the wrong charset the em dash is skipped.
There are two alternatives:
1. Substitute the em dash with a regular hyphen.
2. Change the test to expect the wrong text, with the em dash omitted.

Is the charset wrong on purpose?

Reply:
I didn't change this part in the new version.